addons/website_event_track/static/src/js/website_event_track.js

```diff
@@ -96,3 +96,16 @@ publicWidget.registry.websiteEventTrack = publicWidget.Widget.extend({
         }
     },
 });
+
+publicWidget.registry.eventTrackForm = publicWidget.Widget.extend({
+    selector: '.o_wevent_event_search_box_track',
+    events: {
+        'search .search-query': '_onClearSearchInput',
```

Comment:
This event is called whenever the search is executed and not only when clearing the input.
So I think the method should be called `_onSearch` like for the basic website search box.